File: source/Emphasize/Format Definitions/Bold.vb
Imports Microsoft.VisualStudio.Text.Classification
Imports Microsoft.VisualStudio.Utilities
Imports System.ComponentModel.Composition


Namespace FormatDefinitions

    <ClassificationType(ClassificationTypeNames:=Bold.Name)>
    <Export(GetType(EditorFormatDefinition))>
    <Name(Bold.Name)>
    <Order(After:=Priority.High)>
    <UserVisible(True)>
    Public NotInheritable Class Bold
        Inherits ClassificationFormatDefinition


        Public Const Name As String = "Comment - Bold Span"


        <Export(GetType(ClassificationTypeDefinition))>
        <BaseDefinition("Comment")>
        <Name(Name)>
        Public Shared ReadOnly ClassificationType As ClassificationTypeDefinition


        Public Sub New()
            DisplayName = Name
            IsBold = True
        End Sub

    End Class

End Namespace

File: source/Emphasize/FormatDefinitions/Bold.cs
#nullable enable

using Microsoft.VisualStudio.Text.Classification;
using Microsoft.VisualStudio.Utilities;
using System.ComponentModel.Composition;


namespace Emphasize.FormatDefinitions {

    [ClassificationType(ClassificationTypeNames = Name)]
    [Export(typeof(EditorFormatDefinition))]
    [Name(Name)]
    [Order(After = Priority.High)]
    [UserVisible(true)]
    public class Bold : ClassificationFormatDefinition {

        public const string Name = "Emphasize - Bold";


        [Export(typeof(ClassificationTypeDefinition))]
        [Name(Name)]
        public static readonly ClassificationTypeDefinition? ClassificationType;


        public Bold() {
            DisplayName = Name;
            IsBold = true;
        }

    }

}
